modules/users/client/controllers/settings/manage-social-accounts.client.controller.js

```diff
@@ -7,7 +7,9 @@ angular.module('users').controller('SocialAccountsController', ['$scope', '$http
     // Check if there are additional accounts
     $scope.hasConnectedAdditionalSocialAccounts = function (provider) {
       for (var i in $scope.user.additionalProvidersData) {
-        return true;
```

Comment:
What this is checking for is whether or not the User has any data in the `additionalProvidersData` field. Unfortunately, it's not an array so we can't just check for length. This function should really be just one line:
`return Object.keys($scope.user.additionalProvidersData).length;`